newIDE/app/src/AssetStore/PrivateAssets/PrivateAssetPackPurchaseDialog.js

```diff
@@ -0,0 +1,291 @@
+        content: (
+          <Line justifyContent="center" alignItems="center">
+            <Text>
+              <Trans>
+                You can close this window and go back to the asset store to
+                download your assets.
+              </Trans>
+            </Text>
+          </Line>
+        ),
+      }
+    : isBuying
+    ? {
+        subtitle: <Trans>Complete your payment on the web browser</Trans>,
+        content: (
+          <>
+            <Line justifyContent="center" alignItems="center">
+              <CircularProgress size={20} />
+              <Spacer />
+              <Text>Waiting for the purchase confirmation...</Text>
+            </Line>
+            <Spacer />
+            <Line justifyContent="center">
+              <BackgroundText>
+                <Trans>
+                  Once you're done, come back to GDevelop and the assets will be
+                  added to your account automatically.
+                </Trans>
+              </BackgroundText>
+            </Line>
+          </>
+        ),
+      }
+    : isCheckingPurchasesAfterLogin
+    ? {
+        subtitle: <Trans>Loading your profile...</Trans>,
+        content: (
+          <Line justifyContent="center" alignItems="center">
+            <CircularProgress size={20} />
+          </Line>
+        ),
+      }
+    : {
+        subtitle: (
+          <Trans>
+            The Asset Pack {privateAssetPackListingData.name} will be linked to
+            your account {profile.email}
+          </Trans>
+        ),
+        content: (
+          <Line justifyContent="center" alignItems="center">
+            <Text>
+              <Trans>
+                A new secure window will open to complete the purchase.
+              </Trans>
+            </Text>
+          </Line>
+        ),
+      };
+
+  const showContinueButton =
+    profile &&
+    !isBuying &&
+    !purchaseSuccessful &&
+    !isCheckingPurchasesAfterLogin;
+  const dialogActions = [
+    <FlatButton
+      key="cancel"
+      label={purchaseSuccessful ? <Trans>Accept</Trans> : <Trans>Cancel</Trans>}
+      onClick={onClose}
+    />,
+    showContinueButton ? (
+      <DialogPrimaryButton
+        key="continue"
+        primary
+        label={<Trans>Continue</Trans>}
+        onClick={onContinue}
+      />
+    ) : null,
+  ];
+
+  return (
+    <Dialog
+      title={<Trans>{privateAssetPackListingData.name}</Trans>}
+      maxWidth="sm"
+      open
+      onRequestClose={onClose}
+      actions={dialogActions}
+      onApply={purchaseSuccessful ? onClose : onContinue}
+      cannotBeDismissed // Prevent the user from continuing by clicking outside.
+      flexColumnBody
+    >
+      <Line justifyContent="center" alignItems="center">
+        {purchaseSuccessful && (
+          <>
+            <VerifiedUser />
+            <Spacer />
+          </>
+        )}
+        <Text>
+          <b>{dialogContents.subtitle}</b>
```

Comment:
You whould use the appropriate `size` from the Text component rather than using a b tag. Or is it that Text does not allow it?
Or maybe that's how we should do it with Material ui.